meta-oe/recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -0,0 +1,14 @@
+do_install:append () {
+  if ${@bb.utils.contains("DISTRO_FEATURES", "hiba", "false", "true", d)} ; then
+    echo "DISTRO_FEATURES does not support hiba. Skip the installations"
+    exit 0
+  fi
+
+  install -d ${D}/usr/include/ssh
+  install -d ${D}/usr/include/ssh/openbsd-compat
+  install -d ${D}/usr/lib/
+  install -m0644 ${S}/*.h ${D}/usr/include/ssh
+  install -m0644 ${S}/libssh* ${D}/usr/lib/
+  install -m0644 ${S}/openbsd-compat/*.h ${D}/usr/include/ssh/openbsd-compat
+  install -m0644 ${S}/openbsd-compat/libopenbsd-compat* ${D}/usr/lib/
```

Comment:
where are these files coming from ?

Reply:
These are all of the files that's build from https://github.com/openssh/openssh-portable, or do you mean something else?